spec/features/idv/doc_auth/verify_step_spec.rb

```diff
@@ -119,23 +119,6 @@
     expect(page).to have_current_path(idv_doc_auth_verify_step)
   end
 
-  context 'resolution proofing raises a timeout exception' do
-    before do
-      allow_any_instance_of(Proofing::Mock::StateIdMockClient).to receive(:execute_proof).and_raise(
-        Proofing::TimeoutError.new(
-          'ExceptionId: 0047, ExceptionText: MVA did not respond in a timely fashion',
-        ),
-      )
-    end
-
-    it 'does not proceed to the next page if resolution raises a timeout exception' do
-      click_idv_continue
-
-      expect(page).to have_current_path(idv_doc_auth_verify_step)
-      expect(page).to have_content(t('idv.failure.timeout'))
-    end
-  end
```

Comment:
should we keep the test but maybe change the expected error? this should be some sort of retryable failure now right?